.github/workflows/build-test.yaml

```diff
@@ -2623,21 +2623,23 @@ jobs:
             exit 1
           fi
 
-          if [ "$(kubectl get all -n postgres-test --ignore-not-found | wc -l | tr -d '\n')" == "0" ]; then
-            printf "postgres-test namespace does not contain any resources\n\n"
-            kubectl get all -n postgres-test
-            exit 1
-          fi
+          for ns in "${APP_NAMESPACES[@]}"; do
+            if [ "$(kubectl get all,secrets,configmap,pvc -n "$ns" -l "kots.io/app-slug=$APP_SLUG" --ignore-not-found | wc -l | tr -d '\n')" == "0" ]; then
+              printf "%s namespace does not contain any resources\n\n" "$ns"
+              kubectl get all,secrets,configmap,pvc -n "$ns"
+              exit 1
+            fi
+          done
```

Comment:
running through the namespaces as listing all namespace resources is vauge

Reply:
why is this necessary? we already check for remaining resources above.